README.md

```
@@ -0,0 +1,128 @@
+
+
+## Technical and Machine Learning Advice
+* [How becoming not a data scientist made me a better data scientist](https://docs.google.com/presentation/d/1jk-qrVKCb0-P9P4BVzH75gcVhp5Dy5n1CP_gKnHMNY0/edit#slide=id.p) - Coding advice for data scientists by Joel Grus.
+* [Machine Learning: The High Interest Credit Card of Technical Debt ](https://ai.google/research/pubs/pub43146) - The famous Google paper on technical debt in machine learning systems.
+* [Rules of Machine Learning: Best Practices for ML Engineering](http://martin.zinkevich.org/rules_of_ml/rules_of_ml.pdf) - 43 rules for implementing ML products.
+* [Software Engineering for Machine Learning: A Case Study](https://www.microsoft.com/en-us/research/publication/software-engineering-for-machine-learning-a-case-study/) - Survey results from Microsoft's AI practicioners, including best practices.
+* [Software Engineering for Data Scientists - Small Big Data: large data on a single computer](https://pythonspeed.com/datascience) - Several guides on how to reduce memory usage with e.g. pandas.
+* [Coding habits for data scientists](https://www.thoughtworks.com/insights/blog/coding-habits-data-scientists) - From messy notebook prototyping to production-ready code. The author also created [Clean Code concepts adapted for machine learning and data science](https://github.com/davified/clean-code-ml), including a [video series](https://www.youtube.com/watch?v=Edn6XxWmtEs&list=PLO9pkowc_99ZhP2yuPU8WCfFNYEx2IkwR&index=2).
+* [How to pick more beautiful colors for your data visualizations](https://blog.datawrapper.de/beautifulcolors/) – Common color mistakes and how to avoid them.
+* [Blockchain, the amazing solution for almost nothing ](https://thecorrespondent.com/655/blockchain-the-amazing-solution-for-almost-nothing)
+* [https://github.com/zakirullin/cognitive-load](Cognitive Load is what matters) – Optimize your code to reduce cognitive load
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)